src/main/java/com/vinyl/service/Validator.java

```diff
@@ -1,8 +1,6 @@
 package com.vinyl.service;
 
-import com.vinyl.model.User;
+public interface Validator<T> {
 
-public interface Validator {
-
-	public void validate(User user);
+	public void validate(T obj);
```

Comment:
method is already declared as public in interfaces, no need to add it explicitly

Reply:
fixed